plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -66,11 +69,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define info(format, ...) do_log(LOG_INFO, format, ##__VA_ARGS__)
 #define debug(format, ...) do_log(LOG_DEBUG, format, ##__VA_ARGS__)
 
+#define INTEL_VENDOR_ID 0x8086;
```

Comment:
I would not recommend making a magic value an external linkage. Its much easier to find an missing include than diagnose a linker errors.

Everywhere its used should be safe to just include `common_utils.h` (and actually already do include it) so that seems like a good place for it.

Reply:
Given that this is C++ and we require C++17, a native variant would be:

```
constexpr inline int INTEL_VENDOR_ID = 0x8086;
```

which can be used in a header and is not an ODR resolution, as the value doesn't change.

Magic numbers are bad™, but at least C++ gives us proper language constructs to use them without relying purely on textual replacement.

Reply:
This was changed to:
```
constexpr inline int INTEL_VENDOR_ID = 0x8086;
```
in `plugins/obs-qsv11/common_utils.h` as requested.